mlir/utils/jenkins/Jenkinsfile

```diff
@@ -281,6 +281,29 @@ pipeline {
                                       """
                             }
                         }
+                        stage("Test MIOpen config with tuning") {
+                            when {
+                                    equals expected: true, actual: params.weekly;
+                            }
+                            environment {
+                                // Make libMIOpen.so accessible to the test driver
+                                LD_LIBRARY_PATH="${WORKSPACE}/MIOpen/build/lib:$LD_LIBRARY_PATH:"
+                            }
+                            steps {
+                                sh 'ldconfig'
+                                dir('MIOpen/build/') {
+                                    sh 'date --utc +%Y-%m-%d > tuning-date'
+                                    sh """
+                                    bash ${WORKSPACE}/mlir/utils/jenkins/miopen-tests/miopen_validate.sh --tune-all\
+                                       < ${WORKSPACE}/mlir/utils/jenkins/miopen-tests/resnet50-miopen-configs"""
+                                    sh 'ls -l MIOpenUserDB'
+                                }
+                                // Save user database  for nightly jobs
+                                archiveArtifacts artifacts: 'MIOpen/build/MIOpenUserDB/**, MIOpen/build/tuning-date',\
```

Comment:
1) This only runs weekly, right?

2) Have you tested to make sure this won't conflict with performance report archiving?

Reply:
Yes. This runs weekly. I haven't tested this with performance reports yet, but I don't think they will conflict, as they are different CI jobs.

Reply:
Yeah, that's true, but currently `weekly` duplicates a lot of `nightly` and so we'll be getting things like weekly performance regressions.